Corrfunc/mocks/DDtheta_mocks.py

```diff
@@ -351,6 +351,304 @@ def DDtheta_mocks(autocorr, nthreads, binfile,
+            DDtheta_mocks_extn
+    except ImportError:
+        msg0 = "Could not import the C extension for the angular "\
+              "correlation function for mocks."
+        raise ImportError(msg0)
+
+    import numpy as np
+    from Corrfunc.utils import translate_isa_string_to_enum, fix_ra_dec,\
+        return_file_with_rbins, convert_to_native_endian, process_weights
+    from future.utils import bytes_to_native_str
+    import itertools
+    import time
+
+    weights1, weights2 = process_weights(weights1, weights2, RA1, RA2,
+                                         weight_type, autocorr)
+
+    # Ensure all input arrays are native endian
+    RA1, DEC1, weights1, RA2, DEC2, weights2 = [
+        convert_to_native_endian(arr, warn=True) for arr in
+        [RA1, DEC1, weights1, RA2, DEC2, weights2]]
+
+    fix_ra_dec(RA1, DEC1)
+    if autocorr == 0:
+        fix_ra_dec(RA2, DEC2)
+
+    # Passing None parameters breaks the parsing code, so avoid this
+    kwargs = {}
+    for k in ['weights1', 'weights2', 'weight_type', 'RA2', 'DEC2']:
+        v = locals()[k]
+        if v is not None:
+            kwargs[k] = v
+
+    integer_isa = translate_isa_string_to_enum(isa)
+    rbinfile, delete_after_use = return_file_with_rbins(binfile)
+    bin_refs = np.arange(1, maxbinref + 1)
+    if link_in_ra:
+        bin_ref_perms = itertools.product(bin_refs, bin_refs)
+    else:
+        bin_ref_perms = [(1, binref) for binref in bin_refs]
+    dtype = np.dtype([(bytes_to_native_str(b'nRA'), np.int),
+                      (bytes_to_native_str(b'nDEC'), np.int),
+                      (bytes_to_native_str(b'avg_time'), np.float),
+                      (bytes_to_native_str(b'sigma_time'), np.float)])
+    all_runtimes = np.zeros(maxbinref ** 3, dtype=dtype)
+    all_runtimes[:] = np.inf
+
+    if autocorr == 0:
+        if RA2 is None or DEC2 is None:
+            msg1 = "Must pass valid arrays for RA2/DEC2 for "\
+                  "computing cross-correlation."
+            raise ValueError(msg1)
+    else:
+        RA2 = np.empty(1)
+        DEC2 = np.empty(1)
+
+    if link_in_ra is True:
+        link_in_dec = True
+
+    if link_in_dec is False and link_in_ra is False:
+        msg2 = "Warning: Brute-force calculation without any gridding "\
+               "is forced, as link_in_dec and link_in_ra are both set "\
+               "to False. Please be sure to turn on at least link_in_dec, "\
+               "or both link_in_dec and link_in_ra on."
+        raise ValueError(msg2)
+
+    if link_in_ra:
+        pass
+
+    if link_in_dec is True:
+        if link_in_ra is False:
+            msg3 = "Info: Gridding in the declination only, as link_in_dec "\
+                   "is set to True while link_in_ra is set to False. "\
+                   "Thus looping is only needed over the range of "\
+                   "(min, max) bin, with refinements in the declination."
+            print(msg3)
```

Comment:
You can just name this variable as `msg`. Do you mind tweaking the wording slightly, and then only printing the message when `verbose=True`, like so:

```python
    if verbose:
            msg = "INFO: Since ``link_in_ra`` is not set, only gridding in declination "\
                       "Checking with refinements in declination ranging from [1, {}] and a "\
                       "maximum of {} bins".format(maxbinref, max_cells_per_dim)
            print(msg)

```